components/concord/common/constants.ts

```diff
@@ -0,0 +1,34 @@
+export const AGREEMENT_STATUS_OPTIONS = [
+  "DRAFT",
+  "TEMPLATE",
+  "CONTRACT",
+];
+
+export const AGREEMENT_PATCH_STATUS_OPTIONS = [
+  "SIGNING",
+  "BROKEN",
+];
+
+export const AGREEMENT_PARAMETERS_SOURCE = [
+  "NONE",
+  "EXCEL",
+  "SALESFORCE",
+];
+
+export const AGREEMENT_LIST_STATUSES = [
+  "DRAFT",
+  "VALIDATION",
+  "NEGOTIATION",
+  "SIGNING",
+  "UNKNOWN_CONTRACT",
+  "FUTURE_CONTRACT",
+  "CURRENT_CONTRACT",
+  "COMPLETED_CONTRACT",
+  "COMPLETED_CANCEL_CONTRACT",
+  "COMPLETED_CONTRACT_RENEWABLE",
+  "BROKEN",
+  // "TRASHED", // listed in the API documentation, but when included the endpoints returns no results
```

Comment:
Since it's available in the docs, I believe that it should be possible to the customer to select it, regardless of the functionality

Reply:
This can be selected as a multi-option prop, so even if many other statuses are selected, when this one is included all of them stop working. So no events are emitted for the source.

Do you think it should still be enabled, and should there be a note mentioning this issue? Otherwise we may have many confused users with their source not emitting events. Also, if no options are selected, the component treats that as "all options", so this should probably be all except this bugged one, right?

Reply:
thank you for the detailed explanation, you are right in my opinion, let's keep it as it is